examples/nodejs-simple/gameserver-counts-and-lists.yaml

```diff
@@ -0,0 +1,46 @@
+---
+# Copyright 2020 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: agones.dev/v1
+kind: GameServer
+metadata:
+  generateName: simple-game-server-
+spec:
+  ports:
+    - name: default
+      portPolicy: Dynamic
+      containerPort: 7654
+  counters:
```

Comment:
I made this to test with the local server but neither the counter or list can be found, so perhaps the structure is incorrect?
I started with
```
sdk-server.windows.amd64.exe --local --feature-gates CountsAndLists=true -f gameserver-counts-and-lists.yaml 
```

Reply:
Thank you, that's what I needed, so can perhaps remove this file and update the example